fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeTypes.java

```diff
@@ -2,56 +2,117 @@
 
 package com.starrocks.privilege;
 
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
+import java.util.HashMap;
+import java.util.Map;
 
 public enum PrivilegeTypes {
```

Comment:
why the name of enum always has an 's' in the end, it's a type not a collection i think.

Reply:
Usted tiene razon.